### ISSUE TITLE
Feature?

### ISSUE THREAD

**Issue by KennyGoers** [JuniperPhoton/SplitViewEx#1]
You working on this? :-) Started working on my own and ran across you...

**Comment by JuniperPhoton** [comment]
Yep, but Justin seems to have done this in his own way, check [this](https://github.com/JustinXinLiu/SwipeableSplitView ) out :-)

**Comment by KennyGoers** [comment]
How does a XAML / Windows Universal library help?

**Comment by JuniperPhoton** [comment]
Actually, SplitView is no need, which means that you can create your own Swipe-able Navigation drawer in your 8.1 or earlier project. Using XAML to make your job easier.
